pkgs/servers/web-apps/engelsystem/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv
+, fetchzip
+, php
+}:
+
+  stdenv.mkDerivation rec {
+    pname = "engelsystem";
+    version = "3.1.0";
+
+     src = fetchzip {
+      url = "https://github.com/engelsystem/engelsystem/releases/download/v3.1.0/engelsystem-v3.1.0.zip";
+      sha256 = "01wra7li7n5kn1l6xkrmw4vlvvyqh089zs43qzn98hj0mw8gw7ai";
+      extraPostFetch = "chmod -R a-w $out";
+    };
+
+    buildInputs = [ php ];
```

Comment:
Where is php used?

Reply:
it is used in the bin/migrate script

Reply:
I see, it is being used by the patchShebangs in fixupPhase